tools/megablast_xml_parser/megablast_xml_parser.py

```diff
@@ -1,78 +1,71 @@
 #!/usr/bin/env python
-    
-import sys, os, re
 
-if sys.version_info[:2] >= ( 2, 5 ):
-    import xml.etree.cElementTree as ElementTree
-else:
-    from galaxy import eggs
-    import pkg_resources; pkg_resources.require( "elementtree" )
-    from elementtree import ElementTree
+import re
+import sys
+import xml.etree.cElementTree as ElementTree
 
-def stop_err( msg ):
-    sys.stderr.write( "%s\n" % msg )
+
+def stop_err(msg):
+    sys.stderr.write("%s\n" % msg)
     sys.exit()
```

Comment:
Just `sys.exit("%s\n" % msg") `

Reply:
I think ``sys.exit(msg)`` should work just as well:

```
$ python -c "import sys; sys.exit('Bye')"
Bye
```

Reply:
> I think `sys.exit(msg)` should work just as well:

Yes, and there's no need to define a `stop_err` function to wrap that.